### ISSUE TITLE
Not OK: perl v5.8.1 +MAINT20844 on MSWin32-x86-multi-thread 4.0 (UNINSTALLED)

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#6727]
Migrated from [rt.perl.org#23579](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=23579) (status was 'resolved')

Searchable as RT23579$

**Comment by p5pRT** [comment]
### From andreas@mrs.ch



MingW32\, NT4\, SP6a

Failed Test                      Stat Wstat Total Fail  Failed  List of Failed
\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-
\.\./ext/Devel/DProf/t/DProf\.t                   20   15  75\.00%  1\-8 13\-17 19\-20
\.\./lib/Net/Ping/t/190\_alarm\.t       9  2304     6    1  16\.67%  6
\.\./lib/Net/Ping/t/450\_service\.t                26    4  15\.38%  8\-10 22
\.\./lib/Net/Ping/t/510\_ping\_udp\.t                2    1  50\.00%  2
io/dup\.t                                       12    2  16\.67%  6\-7
io/open\.t                                     100    2   2\.00%  35\-36
io/openpid\.t                                   10    1  10\.00%  9
win32/system\.t                               1008   12   1\.19%  225 253 281 309
&nbsp;                                                                561\-562 589
&nbsp;                                                                617\-618 645 785
&nbsp;                                                                813
 \(1 subtest UNEXPECTEDLY SUCCEEDED\)\, 55 tests and 663 subtests skipped\.
Failed 8/813 test scripts\, 99\.02% okay\. 38/76887 subtests failed\, 99\.95% okay\.
dmake\.exe&#8203;:  Error code 255\, while making 'test'

F&#8203;:\\temp\\perl@&#8203;20845\\win32>

<details><summary>Perl Info</summary>

```

Flags:
    category=install
    severity=none

Site configuration information for perl v5.8.1:

Configured by FehrA at Mon Aug 25 10:42:24 2003.

Summary of my perl5 (revision 5 version 8 subversion 1) configuration:
  Platform:
    osname=MSWin32, osvers=4.0, archname=MSWin32-x86-multi-thread
    uname=''
    config_args='undef'
    hint=recommended, useposix=true, d_sigaction=undef
    usethreads=undef use5005threads=undef useithreads=define usemultiplicity=define
    useperlio=define d_sfio=undef uselargefiles=define usesocks=undef
    use64bitint=undef use64bitall=undef uselongdouble=undef
    usemymalloc=n, bincompat5005=undef
  Compiler:
    cc='gcc', ccflags ='-g -O2 -DWIN32 -DHAVE_DES_FCRYPT -DPERL_IMPLICIT_CONTEXT -DPERL_IMPLICIT_SYS -fno-strict-aliasing -DPERL_MSVCRT_READFIX',
    optimize='-g -O2',
    cppflags='-DWIN32'
    ccversion='', gccversion='', gccosandvers=''
    intsize=4, longsize=4, ptrsize=4, doublesize=8, byteorder=1234
    d_longlong=undef, longlongsize=8, d_longdbl=define, longdblsize=12
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='long long', lseeksize=8
    alignbytes=8, prototype=define
  Linker and Libraries:
    ld='gcc', ldflags ='-g -L"c:\bin\temp\perl\5.8.1\lib\MSWin32-x86-multi-thread\CORE" -L"c:\bin\fsf\MingW32\lib"'
    libpth=c:\bin\fsf\MingW32\lib
    libs= -lmsvcrt -lmoldname -lkernel32 -luser32 -lgdi32 -lwinspool -lcomdlg32 -ladvapi32 -lshell32 -lole32 -loleaut32 -lnetapi32 -luuid -lwsock32 -lmpr -lwinmm -lversion -lodbc32
    perllibs= -lmsvcrt -lmoldname -lkernel32 -luser32 -lgdi32 -lwinspool -lcomdlg32 -ladvapi32 -lshell32 -lole32 -loleaut32 -lnetapi32 -luuid -lwsock32 -lmpr -lwinmm -lversion -lodbc32
    libc=-lmsvcrt, so=dll, useshrplib=yes, libperl=libperl58.a
    gnulibc_version='undef'
  Dynamic Linking:
    dlsrc=dl_win32.xs, dlext=dll, d_dlsymun=undef, ccdlflags=' '
    cccdlflags=' ', lddlflags='-mdll -g -L"c:\bin\temp\perl\5.8.1\lib\MSWin32-x86-multi-thread\CORE" -L"c:\bin\fsf\MingW32\lib"'

Locally applied patches:
    MAINT20844


@INC for perl v5.8.1:
    ..\lib
    c:/docs/source/perl
    F:/temp/perl@20845/lib
    .


Environment for perl v5.8.1:
    CYGWIN=tty
    HOME=M:\
    LANG (unset)
    LANGUAGE (unset)
    LC_ALL=C
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=C:\bin\fsf\MingW32\bin;C:\WINNT4.0\system32;C:\WINNT4.0;C:\bin\perl\5.6.1\bin;C:\bin\perl\5.6.1\bin\MSWin32-x86-multi-thread;c:\bin\fsf\dmake
    PERL5LIB=c:/docs/source/perl
    PERL_BADLANG (unset)
    SHELL (unset)



```
</details>

**Comment by p5pRT** [comment]
### From @steve-m-hay

Resolved&#8203;: perl\-5\.8\.4\+MAINT22869 passes all these tests \(using
MinGW\-3\.1\.0\-1\)\.

**Comment by p5pRT** [comment]
@steve-m-hay - Status changed from 'new' to 'resolved'
